### ISSUE TITLE
(apache-httpd) httpd.conf defaults to port 80 often causing port conflicts

### ISSUE THREAD

**Issue by photodude** [rgra/choco-packages#12]
Many services attempt to default to Port 80 in windows, Skype is on of the most notorious for hog-tieing port 80.

As a best practice, Apache should not be configured to port 80 on install via chocolatey to avoid obvious port conflicts. 

### Expected
IPV4 and IPV6 Listen and ServerName values do not default to obvious port conflict values like port 80

Additionally, it would be helpful to pass a specified port number to the `choco install` of apache-httpd
such as `choco install apache-httpd --port 8585`

### Actual
The Listen and ServerName values in httpd.conf are set to port 80

/cc @rgra @AdmiringWorm @ferventcoder

**Comment by ferventcoder** [comment]
I think you were looking for `--params "'/Port:8585'"`.

However I believe the default of Port 80 is the right answer for Apache. It serves web sites.

Skype on the other hand is in the wrong and should probably be redirected to a different port. Thoughts?

**Comment by photodude** [comment]
I've looked a bit into this, and port 80 is likely correct for default according to RFC 2616 which states TCP port 80 is for Hypertext Transfer Protocol (HTTP) or port 443 for Hypertext Transfer Protocol over SSL/TLS (HTTPS) (RFC 2818).

Skype apparently has gotten better in their current installs which now defaults to choosing a port above 1024 at random as the port for incoming connections. But still, defaults to an optional fallback with ports 80 and 443 if the original port above 1024 becomes unavailable. (they still recommend to set apache/IIS/nginx to alternative ports)

I tried your suggestion `choco install apache-httpd -y --params "'/Port:8888'"` but the port remained set to port 80 (would have been perfect if it worked).

I also note the following two issues in the install
- Issue #7 
- Issue #11
